src/Microsoft.AspNetCore.Razor/Parser/TagHelpers/TagHelperBlockRewriter.cs

```diff
@@ -229,6 +238,15 @@ public static class TagHelperBlockRewriter
                     // Check for attribute start values, aka single or double quote
                     if (i < htmlSymbols.Length && IsQuote(htmlSymbols[i]))
                     {
+                        if (htmlSymbols[i].Type == HtmlSymbolType.DoubleQuote)
+                        {
+                            attributeValueStyle = HtmlAttributeValueStyle.DoubleQuotes;
```

Comment:
Neeed to just do single quotes here. Minimized has to be handled below. Can nuke the double quote part though :smile: